src/mapeo-project.js

```diff
@@ -786,40 +789,43 @@ export class MapeoProject extends TypedEmitter {
         }
       }
 
-      let latitude = lat
-      let longitude = lon
-      let altitude = null
-      const position = observation?.metadata?.position?.coords
-      if (position) {
-        latitude = position.latitude
-        longitude = position.longitude
-        if (position.altitude !== undefined) {
-          altitude = position.altitude
+      const metadataCoords = observation.metadata?.position?.coords
+      const altitude = metadataCoords?.altitude
+
+      /** @type {[number, number] | [number, number, number] | null} */
+      let coordinates = null
+
+      // Prioritize using the observation's `lat` and `lon` fields
+      if (typeof lat === 'number' && typeof lon === 'number') {
+        coordinates =
+          typeof altitude === 'number' ? [lon, lat, altitude] : [lon, lat]
+      } else {
+        // Fall back to using the observation metadata's position if possible
+        if (
+          typeof metadataCoords?.latitude === 'number' &&
+          typeof metadataCoords?.longitude === 'number'
+        ) {
+          coordinates =
+            typeof altitude === 'number'
+              ? [metadataCoords.longitude, metadataCoords.latitude, altitude]
+              : [metadataCoords.longitude, metadataCoords.latitude]
         }
       }
 
-      const coordinates = [longitude, latitude]
-      if (typeof altitude === 'number') {
-        coordinates.push(altitude)
+      /** @type {import('geojson').Feature<import('geojson').Point | null>} */
```

Comment:
mentioned [here](https://github.com/digidem/comapeo-core/pull/1039#discussion_r2127215297) but could've been lost, but may need to use `@types/geojson` and have it installed as an explicit dev dep.

```suggestion
      /** @type {import('@types/geojson').Feature<import('@types/geojson').Point | null>} */
```